llmfoundry/utils/builders.py

```diff
@@ -220,7 +220,7 @@ def build_callback(
 
 def build_logger(name: str, kwargs: Dict[str, Any]) -> LoggerDestination:
     kwargs_dict = {
-        k: v if isinstance(v, str) else om.to_container(v, resolve=True)
+        k: v if isinstance(v, str) or isinstance(v, bool) else om.to_container(v, resolve=True)
```

Comment:
oops, I think this should be different though. I think it should be `k: v if not isinstance(v, (ListConfig, DictConfig)) else om.to_container(v, resolve=True). wdyt? cc @aspfohl cause blame says that you wrote this line

Reply:
lgtm

Reply:
Oof, yea I think it should be something like that. Otherwise we would need to consider other types too (int, float, list, etc)

Should we be resolving earlier (eg in train.py?) @dakinggg? I find myself fighting with typing + omegaconf more than I'd like to admit

Reply:
Good point. I think the `pop_config` call in `train.py` for the `loggers` key should actually just have `convert=True`.

Reply:
There is a `om.resolve(cfg)` at the top of `train.py`